useDrauu: Some tips are missing from the document

### Describe the bug

When the user follows the documentation prompts to install useDrauu, the following error will be displayed.
```
✘ [ERROR] Could not resolve "async-validator"

    node_modules/@vueuse/integrations/index.mjs:2:19:
      2 │ import Schema from 'async-validator';
        ╵                    ~~~~~~~~~~~~~~~~~

  You can mark the path "async-validator" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "axios"

    node_modules/@vueuse/integrations/index.mjs:4:18:
      4 │ import axios from 'axios';
        ╵                   ~~~~~~~

  You can mark the path "axios" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "change-case"

    node_modules/@vueuse/integrations/index.mjs:5:140:
      5 │ ...pascalCase, pathCase, sentenceCase, snakeCase } from 'change-case';
        ╵                                                         ~~~~~~~~~~~~~

  You can mark the path "change-case" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "universal-cookie"

    node_modules/@vueuse/integrations/index.mjs:6:19:
      6 │ import Cookie from 'universal-cookie';
        ╵                    ~~~~~~~~~~~~~~~~~~

  You can mark the path "universal-cookie" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "focus-trap"

    node_modules/@vueuse/integrations/index.mjs:9:32:
      9 │ import { createFocusTrap } from 'focus-trap';
        ╵                                 ~~~~~~~~~~~~

  You can mark the path "focus-trap" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "fuse.js"

    node_modules/@vueuse/integrations/index.mjs:10:17:
      10 │ import Fuse from 'fuse.js';
         ╵                  ~~~~~~~~~

  You can mark the path "fuse.js" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "jwt-decode"

    node_modules/@vueuse/integrations/index.mjs:11:23:
      11 │ import jwt_decode from 'jwt-decode';
         ╵                        ~~~~~~~~~~~~

  You can mark the path "jwt-decode" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "nprogress"

    node_modules/@vueuse/integrations/index.mjs:12:22:
      12 │ import nprogress from 'nprogress';
         ╵                       ~~~~~~~~~~~

  You can mark the path "nprogress" as external to exclude it from the bundle, which will remove this error.

✘ [ERROR] Could not resolve "qrcode"

    node_modules/@vueuse/integrations/index.mjs:13:19:
      13 │ import QRCode from 'qrcode';
         ╵                    ~~~~~~~~

  You can mark the path "qrcode" as external to exclude it from the bundle, which will remove this error.
```

### Reproduction

https://stackblitz.com/edit/vitejs-vite-b2vs4h?file=package.json

### System Info

```Shell
none
```


### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/packages/integrations/useDrauu/index.md
+++ b/packages/integrations/useDrauu/index.md
@@ -6,13 +6,19 @@ category: '@Integrations'
 
 Reactive instance for [drauu](https://github.com/antfu/drauu)
 
+## Install
+
+```bash
+npm i drauu
+```
+
 ## Usage
 
 ```html
 <script setup>
 import { ref } from 'vue'
 import { toRefs } from '@vueuse/core'
-import { useDrauu } from '@vueuse/integrations'
+import { useDrauu } from '@vueuse/integrations/useDrauu'
 
 const target = ref()
 const { undo, redo, canUndo, canRedo, brush } = useDrauu(target)

--- a/packages/integrations/useDrauu/index.ts
+++ b/packages/integrations/useDrauu/index.ts
@@ -1,6 +1,6 @@
 import type { Ref } from 'vue-demi'
 import { ref, watch } from 'vue-demi'
-import type { Brush, Drauu, Options } from 'drauu'
+import type { Brush, Drauu, DrawingMode, Options } from 'drauu'
 import { createDrauu } from 'drauu'
 import type { EventHookOn, MaybeElementRef } from '@vueuse/core'
 import { createEventHook, unrefElement } from '@vueuse/core'
@@ -66,8 +66,10 @@ export function useDrauu(
   watch(brush, () => {
     const instance = drauuInstance.value
 
-    if (instance)
+    if (instance) {
       instance.brush = brush.value
+      instance.mode = brush.value.mode as DrawingMode // bug fix
+    }
   }, { deep: true })
 
   const undo = () => drauuInstance.value?.undo()

--- a/packages/integrations/useDrauu/index.ts
+++ b/packages/integrations/useDrauu/index.ts
@@ -68,7 +68,7 @@ export function useDrauu(
 
     if (instance) {
       instance.brush = brush.value
-      instance.mode = brush.value.mode as DrawingMode // bug fix
+      instance.mode = brush.value.mode as DrawingMode
     }
   }, { deep: true })
 

